flink/src/main/java/org/apache/sedona/flink/Catalog.java

```diff
@@ -95,7 +95,8 @@ public static UserDefinedFunction[] getFuncs() {
                 new Functions.ST_Split(),
                 new Functions.ST_S2CellIDs(),
                 new Functions.ST_GeometricMedian(),
-                new Functions.ST_NumPoints()
+                new Functions.ST_NumPoints(),
+                new Functions.ST_Force3D()
```

Comment:
You might need to provide a default parameter here since the zValue is optional.